compiler/qsc_parse/src/lex/cooked.rs

```diff
@@ -51,6 +51,14 @@ pub(crate) enum Error {
     #[error("unrecognized character `{0}`")]
     #[diagnostic(code("Qsc.Lex.UnknownChar"))]
     Unknown(char, #[label] Span),
+
+    #[error("expected identifier after apostrophe, found {0}")]
+    #[diagnostic(code("Qsc.Lex.IncompleteApostropheIdentifier"))]
+    IncompleteAposIdent(raw::TokenKind, #[label] Span),
+
+    #[error("expected identifier after apostrophe, found EOF")]
+    #[diagnostic(code("Qsc.Lex.IncompleteApostropheIdentifierEof"))]
+    IncompleteAposIdentEof(#[label] Span),
```

Comment:
nit: Can you move these up so all the `Incomplete*` cases are next to each other?